headerid plugin causes error with Python 3.5.3

# Steps to reproduce

```
$ pip install pelican
$ pelican-quickstart
// Accept defaults here
$ git clone https://github.com/getpelican/pelican-plugins.git
$ echo "PLUGIN_PATHS = ['pelican-plugins']" >> pelicanconf.py
$ echo "PLUGINS = ['headerid']" >> pelicanconf.py
// creating a site
$ make DEBUG=1 html
pelican PATH-WITHDRAWN/content -o PATH-WITHDRAWN/output -s PATH-WITHDRAWN/pelicanconf.py  -D
DEBUG: Pelican version: 3.7.1
DEBUG: Python version: 3.5.3
DEBUG: Adding current directory to system path
DEBUG: Temporarily adding PLUGIN_PATHS to system path
DEBUG: Loading plugin `headerid`
DEBUG: Registering plugin `headerid`
CRITICAL: AttributeError: module 'headerid' has no attribute 'register'
Traceback (most recent call last):
  File "PATH-WITHDRAWN/.pyenv/versions/temp/bin/pelican", line 11, in <module>
    sys.exit(main())
  File "PATH-WITHDRAWN/.pyenv/versions/3.5.3/envs/temp/lib/python3.5/site-packages/pelican/__init__.py", line 393, in main
    pelican, settings = get_instance(args)
  File "PATH-WITHDRAWN/.pyenv/versions/3.5.3/envs/temp/lib/python3.5/site-packages/pelican/__init__.py", line 382, in get_instance
    return cls(settings), settings
  File "PATH-WITHDRAWN/.pyenv/versions/3.5.3/envs/temp/lib/python3.5/site-packages/pelican/__init__.py", line 53, in __init__
    self.init_plugins()
  File "PATH-WITHDRAWN/.pyenv/versions/3.5.3/envs/temp/lib/python3.5/site-packages/pelican/__init__.py", line 80, in init_plugins
    plugin.register()
AttributeError: module 'headerid' has no attribute 'register'
Makefile:65: recipe for target 'html' failed
make: *** [html] Error 1
$
```

# Expected response
Blog gets generated without above error message.

# Work-around
I have a small fix which works on my system. Will send it in as a pull request soon.

--- a/headerid/__init__.py
+++ b/headerid/__init__.py
@@ -1,1 +1,1 @@
-from headerid import *
+from .headerid import *

